common/terms.html

```diff
@@ -1,343 +1,348 @@
-    values of <code>@type</code>,
-    and values of <a>terms</a> defined to be <em>vocabulary relative</em>
-    are resolved relative to the <a>vocabulary mapping</a>,
-    not the <a>base IRI</a>.</dd>
-  <dt><dfn>set object</dfn></dt><dd>
-    A <a>set object</a> is a <a>map</a> that has an <code>@set</code> <a>entry</a>.
-    It may also have an <code>@index</code> key, but no other <a>entries</a>.</dd>
-  <dt><dfn data-cite="RDF11-CONCEPTS#dfn-subject" class="preserve">subject</dfn></dt><dd>
-    A <a>subject</a> is a <a>node</a> in a <a>linked data graph</a>
-    with at least one outgoing edge,
-    related to an <a>object</a> node through a <a>property</a>.
-    See <dfn data-cite="RDF11-CONCEPTS#dfn-subject" class="preserve">RDF subject</dfn> in [[RDF11-CONCEPTS]].</dd>
-  <dt><dfn>term</dfn></dt><dd>
-    A <a>term</a> is a short word defined in a <a>context</a>
-    that MAY be expanded to an <a>IRI</a>.
-  </dd>
-  <dt><dfn>term definition</dfn></dt><dd>
-    A term definition is an entry in a <a>context</a>,
-    where the key defines a <a>term</a>
-    which may be used within a <a>map</a>
-    as a key, type, or elsewhere that a string is interpreted as a vocabulary item.
-    Its value is either a string (<dfn data-lt="simple term">simple term definition</dfn>),
-    expanding to an absolute IRI,
-    or an <a>expanded term definition</a>.
-  </dd>
-  <dt class="changed"><dfn>type map</dfn></dt><dd class="changed">
-    An <a>type map</a> is a <a>map</a> value of a <a>term</a>
-    defined with <code>@container</code> set to <code>@type</code>,
-    whose keys are interpreted as <a>IRIs</a>
-    representing the <code>@type</code> of the associated <a>node object</a>;
-    the value MUST be a <a>node object</a>, or <a>array</a> of node objects.
-    If the value contains a <a>term</a> expanding to <code>@type</code>,
-    it's values are merged with the map value when expanding.</dd>
-  <dt><dfn>JSON literal</dfn></dt><dd>
-    A <a>JSON literal</a> is a <a>literal</a> where the associated <a>IRI</a> is <code>rdf:JSON</code>.
-    In the <a>value object</a> representation, the value of <code>@type</code> is <code>@json</code>.
-    JSON literals represent values which are valid JSON [[RFC8259]].
-    See <dfn data-cite="JSON-LD11#dfn-json-datatype" class="preserve">JSON datatype</dfn> in [[JSON-LD11]].
-  </dd>
-  <dt><dfn class="preserve">typed value</dfn></dt><dd>
-    A <a>typed value</a> consists of a value,
-    which is a <a>string</a>,
-    and a type,
-    which is an <a>IRI</a>.</dd>
-  <dt><dfn>value object</dfn></dt><dd>
-    A <a>value object</a> is a <a>map</a> that has an <code>@value</code> <a>entry</a>.</dd>
-  <dt><dfn class="preserve">vocabulary mapping</dfn></dt><dd>
-    The vocabulary mapping is set in the <a>context</a> using the <code>@vocab</code> key
-    whose value MUST be an <a>IRI</a> or <code>null</code>.</dd>
-</dl>
+    <dt><dfn data-cite="INFRA#list" class="preserve">array</dfn></dt><dd>
```

Comment:
Note that this might confuse people... array and lists are different things.

Reply:
Yes, but it was not feasible to change existing uses of list, which are associated with RDF lists and oradered arrays in JSON-LD. Array is also more compatible with general JSON and WebIDL nomenclature.